Packs/ExodusIntelligence/IndicatorFields/indicatorfield-CPE23URI.json

```diff
@@ -0,0 +1,30 @@
+{
+    "associatedToAll": false,
+    "associatedTypes": [
+        "Exodus Intelligence"
+    ],
+    "caseInsensitive": true,
+    "cliName": "cpe23uri",
+    "closeForm": false,
+    "content": true,
+    "editForm": true,
+    "group": 2,
+    "hidden": false,
+    "id": "indicator_cpe23uri",
+    "isReadOnly": false,
+    "locked": false,
+    "name": "CPE23URI",
+    "neverSetAsRequired": false,
+    "openEnded": true,
+    "ownerOnly": false,
+    "required": false,
+    "sla": 0,
+    "system": false,
+    "threshold": 72,
+    "type": "multiSelect",
```

Comment:
This should probably be a grid field if I understand your meaning correctly

Reply:
For some reason, the grid type doesn't get populated. Leaving it as it is until further investigation.